java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/utils/HazelcastUtils.java

```diff
@@ -53,6 +60,36 @@ public static boolean isOldestMember(HazelcastInstance hazelcastInstance) {
         return memberIterator.hasNext() && memberIterator.next().equals(hazelcastInstance.getLocalEndpoint());
     }
 
+    /**
+     * Handler to set the member configuration with specific properties which
+     * are bound to an agent by internal IP.
+     * <p>
+     *     Sets CP priority for member based on the agent's private address
+     *     if provided.
+     * </p>
+     * @param properties
+     * @param config
+     */
+    public static void handlePerAgentConfig(Map<String, String> properties, Config config) {
+        String cpPriorities = properties.get("cp_priority");
```

Comment:
We should make it clearer that this method is open to expansion, and although it currently only covers `cp_priority`, it can cover more in the future. This could be achieved by clarifying further in javadocs, adding comments for "sections" of this method, etc.

Reply:
done [fd63f6c](https://github.com/hazelcast/hazelcast-simulator/pull/2222/commits/fd63f6c3785df759fe726cbecb261a87b3562209)